CONTRIBUTING.md

```diff
@@ -0,0 +1,47 @@
+# Contributing to timescaledb-rails
+
+First off, thanks for investing your time in contributing to timescaledb-rails 🎉
+
+* [Code of Conduct](#code-of-conduct)
+* [Getting Started](#getting-started)
+    * [Issues](#issues)
+    * [Pull Requests](#pull-requests)
+
+## Code of Conduct
+
+Read our [Code of Conduct](./CODE_OF_CONDUCT.md) to keep our community approachable and respectable.
+
+## Getting Started
+
+Contributions are made to this repo via Issues and Pull Requests (PRs), before creating your own search for existing Issues and PRs to avoid duplicates.
```

Comment:
add a comma after `your own`.

An alternative wording could be:

Contributions are made to this repo via Issues and Pull Requests (PRs). Please, review existing Issues and PRs before creating your own.